isic/core/storages/s3.py

```diff
@@ -0,0 +1,37 @@
+from datetime import datetime, timedelta
+from urllib.parse import urlencode
+
+from django.utils.encoding import filepath_to_uri
+from storages.backends.s3 import S3Storage
```

Comment:
Note, upstream `django-storages` has renamed this class (though the old names are still supported).